admin/sql/create_indexes.sql

```diff
@@ -7,4 +7,6 @@ CREATE UNIQUE INDEX data_sha256_ndx_lowlevel ON lowlevel (data_sha256);
 CREATE INDEX mbid_ndx_highlevel ON highlevel (mbid);
 CREATE INDEX build_sha1_ndx_highlevel ON lowlevel (build_sha1);
 
+CREATE INDEX lower_musicbrainz_id_ndx_user ON "user" (lower(musicbrainz_id));
```

Comment:
I'm not sure if I should also make this a unique constraint - would it be possible to add the same user twice with different case?  MusicBrainz should never send us this, but it should be taken care of